### ISSUE TITLE
Insert Screen Recording - Crash (Frames not found)

### ISSUE THREAD

**Issue by LazyEti** [NickeManarin/ScreenToGif#1288]
**Describe the bug**
Hello, it seems like the "Insert Screen Recording" feature is not working at all on my end.

**To Reproduce**
1. Create a recording.
2. Add a new recording to it (Insert Screen Recording)
3. When you end the 2nd recording, an error window appears saying that the frames files can't be found.

**Expected behavior**
No crash :')

**Screenshots**
![error](https://github.com/NickeManarin/ScreenToGif/assets/39376613/91beddc7-6d76-4185-8e4a-9b358bf83073)
![noFrames](https://github.com/NickeManarin/ScreenToGif/assets/39376613/2409296e-5581-4a47-b595-d046df2f1027)

**Desktop (please complete the following information):**
 - OS: Win 10.0.17763
 - V. 2.40.1

**Additional context**
I tried looking up the path and there's indeed no temp frames in the folder.
Tested on version 2.37.2 and 2.40.1 but the issue is happening on both.
Thanks!

**Comment by NickeManarin** [comment]
I'm unsure what's causing the frame to be missing.

Have you tried setting the cache folder in the exception list of your Anti-virus?

![image](https://github.com/NickeManarin/ScreenToGif/assets/14798947/c7c4d4e7-8a72-438f-85b8-59706e7be020)

Or perhaps you can try using the memory cache option in Options > Recorder

![image](https://github.com/NickeManarin/ScreenToGif/assets/14798947/af0b5537-2418-456a-ad58-ffdae45f5dcd)

**Comment by pawlos** [comment]
@NickeManarin Actually I think the issue occurs when the memory cache is selected. In that case there are no files stored on the disk but the code assumes there are and they to move them and the exception happens. I'll try to see if there's a solution to that

**Comment by LazyEti** [comment]
Hey @NickeManarin sorry I've just seen this.
I was already using the Memory Cache option but swapping to "Save to file" fixed the issue!
Thanks a bunch for the tip

@pawlos it seems like you're right, the software is trying to lookup the frame path and import them before they've been transferred from the cache file to the disk: 
![image](https://github.com/NickeManarin/ScreenToGif/assets/39376613/bc85a8d2-4b11-44c3-9008-5b579243380c)
Still happening in 2.41 btw

**Comment by pawlos** [comment]
@LazyEti yeah. I've tried looking at it to see if that's an easy fix but I think that's a bigger change at least with my knowledge of the system.
